src/shammodels/gsph/src/modules/ExternalForces.cpp

```
@@ -0,0 +1,426 @@
+    set_constant_G.evaluate();
+    set_constant_c.evaluate();
+
+    if (add_ext_forces_seq.size() > 0) {
+        shamrock::solvergraph::OperationSequence seq(
+            "Add external forces", std::move(add_ext_forces_seq));
+        seq.evaluate();
+    }
+}
+
+template<class T>
+std::shared_ptr<shamrock::solvergraph::INode> register_constant_set(
```

Cppcheck (2.21.0)

[style] The function 'register_constant_set' is never used.

(unusedFunction)